style.css

```diff
@@ -260,6 +314,48 @@ nav {
     background-color: #5765dd;
   }
 
+  .title-button h2 {
+    text-align: left;
+  }
+
+  .modal-image img {
+    display: flex;
+    justify-content: space-around;
+    height: 10vh;
+    width: 5vw;
+  }
+
+  .close-btn {
+    float: right;
+    padding-bottom: 1.25rem;
+  }
+
+  .modal-button {
+    display: none;
+  }
+
+  .mobile-buttons {
+    display: flex;
+  }
+
+  .mobile-button {
```

Comment:
- [ ] The buttons on the mobile version are not correctly aligned inside their container. Please decrease the space to the sides as it is looking a bit bigger when compared to the design. Please also make sure the contents of the buttons are on a single line. 🧐️

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://github.com/NewtonOmbese/Newton-Portfolio/assets/17243325/89cdb076-c806-4583-8e9e-e8cbdc7d99ea" width="300">|<img src="https://github.com/NewtonOmbese/Newton-Portfolio/assets/17243325/73cf797c-3838-4fa7-8488-207dbcfc2df5" width="300">|